src/fileSystem.ts

```diff
@@ -32,11 +32,31 @@ export async function writeJsonToPath({
   path: relativePath,
   data,
 }: WriteDataToPathInput) {
+  await writeFileToPath({
+    path: relativePath,
+    content: JSON.stringify(data, null, 2),
+  });
+}
+
+interface WriteContentToPathInput {
+  path: string;
+  content: string;
+}
+
+export async function writeFileToPath({
+  path: relativePath,
+  content,
+}: WriteContentToPathInput) {
   const directory = getRootStorageDirectory();
   const fullPath = path.resolve(directory, relativePath);
 
   await ensurePathCanBeWrittenTo(fullPath);
-  await fs.writeFile(fullPath, JSON.stringify(data, null, 2), 'utf8');
+  await fs.writeFile(fullPath, content, 'utf8');
+}
+
+export async function readJsonFile<T>(path: string) {
```

Comment:
How about `readJsonFromPath`, to balance the name with `writeJsonToPath`? For both functions, I wonder about dropping the `ToPath` bit from the function name since the input object has function calls looking like `writeJson({ path: '/somewhere', content: dataToStringify })`.

Reply:
I can make the first change you suggested. I understand what you are saying about dropping the `toPath` piece I would be fine with removing it. I think it does make the code a bit cleaner.